### ISSUE TITLE
Coronavirus duration on surfaces

### ISSUE THREAD

**Issue by tkhans** [flattenthecurve/guide#68]
How long coronavirus last on surfaces of all kinds. Plastic, Metal, Clothes, and more important metals that we regularly touch. 
Lift buttons
Mobile phones
Laptops
Doors
Faucets
Clothing
Railing
Plastic ware
Kitchen ware
etc.

a chart showing the above in the flattenthecurve.com will be beneficial. 

for each and every important information a chart/picture will be worth a thousand words.

**Comment by jmcmurry** [comment]
Agree. Side note, where is the data about longevity of this particular strain on surfaces? Why was this not the first thing people were studying? In any case, yes we should do the best we can.

**Comment by tkhans** [comment]
https://www.npr.org/sections/health-shots/2020/03/14/811609026/the-new-coronavirus-can-live-on-surfaces-for-2-3-days-heres-how-to-clean-them

there are many such articles that describe different time duration of viruses that last on things. By pulling all such data in a more constructive table will further increase awareness.
